refrepo_git.py

```diff
@@ -283,6 +308,7 @@ def main():
     conf_dir = Path(os.getenv(_CONF_DIR_ENV_VAR, default=_DEFAULT_CONF_DIR))
 
     args = inject_reference_repo_arg(sys.argv[1:], root_dir, repo)
+    add_alternates(sys.argv[1:], root_dir, repo)
```

Comment:
With this addition, is `inject_reference_repo_arg` still necessary?

Reply:
Yes, for git clone I can't modify the alternates file before the clone, since the repo isn't created yet. And after it is too late. Basically the same for submodules.